report_s3.py

```diff
@@ -4,6 +4,7 @@
 import json
 import sys
 
+from pcluster import __version__
```

Comment:
What does this script do?

Reply:
The import allows us to reference the version number of our CDK app. This script generates AWS CLI commands (e.g., "aws s3 cp XXX s3://YYY/ZZZ" which are executed by the calling upload.sh. By embedding the version number here, we get all commands to include it as a prefix on the s3 urls (s3://YYY/__version__/ZZZ).